docs/DeveloperGuide.md

```diff
@@ -671,29 +673,416 @@ testers are expected to do more *exploratory* testing.
       Expected: No customer is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   4. Other incorrect `deletecustomer` commands to try: `deletecustomer`, `deletecustomer x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+2. Deleting a customer from a filtered customer list using the command alias.
+
+    1. Prerequisites: Prerequisites: Customers filtered using `findcustomer` command with at least 1 customer listed.
+       Example: `findcustomer john`
+
+    2. Test case: `deletec 1`<br>
+       Expected: First customer is deleted from the list. Details of the deleted customer shown in the status message.
+
+    3. Test case: `deletec 0`<br>
+       Expected: No customer is deleted. Error details shown in the status message. Status bar remains the same.
+
+### Adding an order
+
+**Note:** Some of the test cases may depend on previous test cases, especially those on testing orders with duplicate/similar names. You are advised to follow the test cases in order. <br>
+
+1. Adding a unique order with all parameters specified while all customers is being shown.
+
+    1. Prerequisites: At least 1 customer is displayed in the customer list.
+        - All orders under a customer are listed using the `listorder 1` command with at least 1 order listed.
+        - There are no orders similar to the order to be added under the first customer.
+
+    2. Test case: `addorder 1 i/Lamp d/20-11-2024 q/3`<br>
+       Expected: The order is successfully added. Details of the added order shown in the status message. All orders associated with the customer are shown in the order list.
+
+2. Adding a unique order with optional field omitted using the command alias when the customer list is filtered.
+
+    1. Prerequisites: Prerequisites: Customers filtered using `findcustomer` command with at least 1 customer listed.
+       Example: `findcustomer bernice`
+
+    2. Test case: `addo 1 i/Books d/02-03-2026` <br>
+       Expected: The order is successfully added with default quantity of `1`. Details of the added order shown in the status message. All orders associated with the customer are shown in the order list.
+
+3. Adding an order with missing compulsory field(s).
+
+    1. Prerequisites: At least 1 customer is displayed in the customer list.
+
+    2. Test case: `addorder 1 i/books` <br>
+
+    3. Expected: No order is added. Error details shown in the status message. Status bar remains the same.
+
+    4. Test case: `addo 1 q/100` <br>
+
+    5. Expected: No order is added. Error details shown in the status message. Status bar remains the same.
+
+4. Adding a [similar order](#similar).
+
+    1. Prerequisites: At least 1 customer is displayed in the customer list. Order similar to the order to be added under the first customer.
```

Comment:
Can change to this just in case people nitpick?
```suggestion
    1. Prerequisites: At least 1 customer is displayed in the customer list. 
    2. There is an existing order similar to the order to be added under the first customer.
```